Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -177,62 +180,69 @@ spec:
-    * `replicasPerFailureDomain`: Sets up the number of replicas to place in a given failure domain. For instance, if the failure domain is a datacenter (cluster is
-stretched) then you will have 2 replicas per datacenter where each replica ends up on a different host. This gives you a total of 4 replicas and for this, the `size` must be set to 4. The default is 1.
-    * `subFailureDomain`: Name of the CRUSH bucket representing a sub-failure domain. In a stretched configuration this option represent the "last" bucket where replicas will end up being written. Imagine the cluster is stretched across two datacenters, you can then have 2 copies per datacenter and each copy on a different CRUSH bucket. The default is "host".
-* `erasureCoded`: Settings for an erasure-coded pool. If specified, `replicated` settings must not be specified. See below for more details on [erasure coding](#erasure-coding).
-    * `dataChunks`: Number of chunks to divide the original object into
-    * `codingChunks`: Number of coding chunks to generate
-* `failureDomain`: The failure domain across which the data will be spread. This can be set to a value of either `osd` or `host`, with `host` being the default setting. A failure domain can also be set to a different type (e.g. `rack`), if the OSDs are created on nodes with the supported [topology labels](../Cluster/ceph-cluster-crd.md#osd-topology). If the `failureDomain` is changed on the pool, the operator will create a new CRUSH rule and update the pool.
+-   `replicated`: Settings for a replicated pool. If specified, `erasureCoded` settings must not be specified.
+    _ `size`: The desired number of copies to make of the data in the pool.
+    _ `requireSafeReplicaSize`: set to false if you want to create a pool with size 1, setting pool size 1 could lead to data loss without recovery. Make sure you are **ABSOLUTELY CERTAIN** that is what you want.
+    _ `replicasPerFailureDomain`: Sets up the number of replicas to place in a given failure domain. For instance, if the failure domain is a datacenter (cluster is
+    stretched) then you will have 2 replicas per datacenter where each replica ends up on a different host. This gives you a total of 4 replicas and for this, the `size` must be set to 4. The default is 1.
+    _ `subFailureDomain`: Name of the CRUSH bucket representing a sub-failure domain. In a stretched configuration this option represent the "last" bucket where replicas will end up being written. Imagine the cluster is stretched across two datacenters, you can then have 2 copies per datacenter and each copy on a different CRUSH bucket. The default is "host".
+-   `erasureCoded`: Settings for an erasure-coded pool. If specified, `replicated` settings must not be specified. See below for more details on [erasure coding](#erasure-coding).
```

Comment:
I wasn't clear from the previous thread... When rendered by mkdocs, do the spaces between the bullet and the first word go away? As long as it looks normal rendered, it's no problem to have this oddity in the markdown.

Reply:
The spaces don't render differently. The spacing is (in theory) to help with multi-paragraph bullet points (which Rook uses several times) not be aligned strangely when the tab stop is 4

So this is what it could look like which is fine but a little awkward to read, and possibly unclear how it will render.
```md
- this has 1 space

    but the next paragraph needs to be indented 4
    
- next bullet
```

```md
-   this has 3 spaces

    and is aligned with the next paragraph
    
-   next bullet
```

Reply:
That local preview is working for you?